### ISSUE TITLE
Feature Request: Castbar Timer Customization Options

### ISSUE THREAD

**Issue by Qwertdd** [Bodify/BetterBlizzPlates#28]
The option [AddOns > BetterBlizzPlates > Enemy Nameplates > Cast timer next to castbar] is a very welcome addition, but right now this feature only results in a huge and un-customizable cast timer directly to the right of the bar.

Would it be possible for this setting to get more in-depth options? Personally, x/y offsets and text size would be enough, but how much attention you give this feature would be up to you.

Thanks!

**Comment by Bodify** [comment]
Yes it is planned! Taking me some time to get around to it though but hopefully soon.

**Comment by Bodify** [comment]
I forgot to mention I did add a variable that controls the text size that you can manually change atm so if you type `/run BetterBlizzPlatesDB.npTargetTextSize = 12` you can change it from its default value of 12 to something else. But yeah I will add more settings in the gui soon enough
